sympy/functions/elementary/tests/test_integers.py

```diff
@@ -560,3 +560,6 @@ def test_nested_floor_ceiling():
     assert ceiling(-floor(ceiling(x**3)/y)) == -floor(ceiling(x**3)/y)
     assert floor(ceiling(-floor(x**Rational(7, 2)/y))) == -floor(x**Rational(7, 2)/y)
     assert -ceiling(-ceiling(floor(x)/y)) == ceiling(floor(x)/y)
+def test__issue_18421():
```

Comment:
```suggestion


def test_issue_18421():
```
This should make use of some spacings